docs/scripting/game-tests.md

```diff
@@ -56,32 +45,67 @@ Beta 1.18.20.21+ (Release 1.18.30+)
 AKA "mojang-minecraft-ui"
 
 **@minecraft/server-admin** - `53d7f2bf-bf9c-49c4-ad1f-7c803d947920`  
-BDS module for configuration of variables and secrets. Cannot be used on Minecraft clients.  
+BDS module for configuration of variables and secrets. This cannot be used on Minecraft clients.  
```

Comment:
Should we write BDS or Bedrock Dedicated Server? It isn't mentioned [here](https://wiki.bedrock.dev/meta/style-guide.html#acronyms-when-discussing) so the readers might not know what it is refferring to